src/System.Threading.Tasks.Dataflow/src/Base/DataflowBlock.cs

```diff
@@ -2653,17 +2653,17 @@ private Task ProcessItemAsync(TOutput item)
                         }
 
                         // If there are SendAsync tasks to wait on...
-                        if (_TempSendAsyncTaskList != null && _TempSendAsyncTaskList.Count > 0)
+                        if (_tempSendAsyncTaskList != null && _tempSendAsyncTaskList.Count > 0)
                         {
                             // Consolidate all SendAsync tasks into one
                             var allSendAsyncTasksConsolidated =
 #if PRENET45
```

Comment:
Yeah, I'm having an offline conversation about that with a few folks. We should either clean this up (removing the compilation constant and associated dead code) or fix the .NET 4 build and give it a project file... right now it's broken.